core/src/test/java/org/infinispan/lock/ExplicitUnlockTest.java

```diff
@@ -140,7 +138,7 @@ private void doRun() throws Exception {
                validateCache();
 
                if (withUnlock) {
-                  unlock(lockKey);
+                  unlock();
```

Comment:
well, I would rather remove the test the change it :) 
we don't support users retrieving the `LockManager`, unlock keys and expect the transaction to commit fine...

Reply:
Why you say that @pruivo?

Even though `LockManager` is not `@public`, it is accessible via the public `AdvancedCache.getLockManager()` method, and there is no warning on the `unlock()` method that it is not supported.

Reply:
well, I don't see any use case where a user needs to acquire/release locks from the `LockManager`. Even if there is a weird use case, I'm pretty sure data consistency is automatically lost. 
Since we have clustered-locks available, I think we could remove the method `AdvancedCache.getLockManager()`

Reply:
I agree `LockManager` is not a good API, but we need to check if some stuff like `getOwner(key)` or `getNumberOfLocksHeld()` should be kept in the API for diagnostics.